federation-integration-testsuite-js/package.json

```diff
@@ -18,8 +18,6 @@
   },
   "homepage": "https://github.com/apollographql/federation#readme",
   "dependencies": {
-    "@opentelemetry/api": "^0.20.0",
```

Comment:
The dependencies of this package doesn't really matter since it is not published.  I think you may want to add it to the dev deps of the top level (also unpublished) package.json so that it's a non peer deps somewhere? But maybe it works if you don't? Perhaps if this works then it's good enough.

Reply:
In theory I guess as long as npm7 is used these dependencies are not needed? api will come in as a peer dependency, the others shouldn't be needed.